qemu/tests/cfg/virt_firmware_check_phys_bits.cfg

```diff
@@ -0,0 +1,52 @@
+- virt_firmware_check_phys_bits:
+    only q35
+    only ovmf
+    type = virt_firmware_check_phys_bits
+    start_vm = no
+    only Linux
+    phys_bits_flags = "host-phys-bits=on,host-phys-bits-limit=%s"
+    phys_bits_grep_cmd = "cat /proc/cpuinfo | grep -m1 'address sizes' | awk '{split($0, arr, " "); print arr[4]}'"
+    limitation_from_ovmf = 46
+    limitation_msg = "limit PhysBits to 46 \(avoid 5-level paging\)"
+    no Host_RHEL.m7 Host_RHEL.m8 Host_RHEL.m9.u0 Host_RHEL.m9.u1 Host_RHEL.m9.u2
+    variants:
+        - with_invalid_value:
+            variants:
+                - _-1:
+                    err_msg = "Parameter 'host-phys-bits-limit' expects uint8_t"
+                    host_phys_bits_limit = -1
+                - _1:
+                    err_msg = "phys-bits should be between 32 and 52  (but is 1)"
+                    host_phys_bits_limit = 1
+        - intel:
```

Comment:
For the valid values, any possibility that we use the same values? if a value(e.g. 39) is specified for intel only, we can add 'only HostCpuVendor.intel' filter, so
- invalid_value
     xxx
- valid_value
     xxx

Reply:
OK, I will update. Thanks.